weave/workers/weave_scorer.py

```diff
@@ -415,21 +450,41 @@ async def _process_monitor(active_monitor: ActiveMonitor) -> None:
             monitor.query,
         )
 
+        sampled_scored_call_ids: list[str] = []
+
         with get_trace_server().call_batch():
-            await asyncio.gather(
-                *[
-                    apply_scorer(
-                        monitor_internal_ref, scorer, call, project_id, wb_user_id
+            scoring_tasks = []
+            for call in calls:
+                # Apply sampling rate
+                if random.random() > monitor.sampling_rate:
```

Comment:
isn't this supposed to be `<=` ?

Reply:
I don't think so.
If your rate is `0.1`, any random pick above 0.1 (i.e. 90% of picks) should result in the call being skipped.